lib/endpoints/class-wp-json-controller.php

```diff
@@ -103,4 +103,21 @@ public function get_item_schema() {
 		return array();
 	}
 
+	/**
+	 * Get an array of endpoint arguments from the item schema for the controller.
+	 * 
+	 * @return array
+	 */
+	public function get_endpoint_args_for_item_schema() {
+
+		$schema                = $this->get_item_schema();
+		$post_type_fields      = ! empty( $schema['properties'] ) ? $schema['properties'] : array();
+		$post_type_fields_args = array();
+
+		foreach ( $post_type_fields as $field_id => $params ) {
+			$post_type_fields_args[$field_id] = array();
```

Comment:
I don't think we have any tests specifically testing that - none of the `args` have any options currently, so no `required` flags to break. Also, I believe if a value is in `$_POST`, it still gets handed to the callback as an arg, even though it wasn't specified in the `args` options.